toqito/nonlocal_games/nonlocal_game.py

```diff
@@ -5,11 +5,74 @@
 
 import cvxpy
 import numpy as np
+import numba as nb
 
 from toqito.helper import npa_constraints, update_odometer
 from toqito.matrix_ops import tensor
 from toqito.rand import random_povm
 
+from binary_constraint_system_game import check_perfect_commuting_strategy,tensor_to_raw
```

Comment:
This file and functions don't exist in your MR. I'm also wondering if these should be defined somewhere in the nonlocal games module?